### DISCUSSION TITLE
Bug Bounty Research Study for Open-Source Maintainers!

### DISCUSSION THREAD

**Discussion by jayala-29** [community/community#102330] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone!

My name is Jessy Ayala, a [PhD student at UC Irvine](https://www.linkedin.com/in/jessy-ayala). Our team has recently submitted a study to a top-tier software engineering conference, and we are curious about the experiences of open-source maintainers who have experience reviewing bug bounty reports.

Your participation will significantly contribute to our research, and we would be grateful for your insights. **If interested, please follow [this link to fill out a brief Likert-scale survey](https://forms.gle/K4YX2HgnUDHJgozh7) which should not take more than a few minutes.**

Thank you for your consideration, your expertise and perspectives are crucial to the success of our study - and to further securing the open-source ecosystem!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @jayala-29 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have closed and locked this Discussion, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
